openmdao/docs/features/building_components/explicitcomp.rst

```diff
@@ -16,6 +16,13 @@ ExplicitComponent Methods
 
 The implementation of each method will be illustrated using a simple explicit component that computes the output *area* as a function of inputs *length* and *width*.
 
+::
```

Comment:
are you talking about the stuff I have here: http://blue.readthedocs.io/en/latest/style_guide/doc_style_guide.html